src/tox/util/pep517/backend.py

```diff
@@ -23,11 +23,11 @@ def __init__(self, backend_module, backend_obj):
     def __call__(self, name, *args, **kwargs):
```

Comment:
Revert changes on this file. This needs to be python 2 compatible to allow python 2 backends. See https://github.com/eumiro/tox/blob/fix-typos/.pre-commit-config.yaml#L20-L21 pinning it so.

Reply:
I was actually wondering why there was so much `.format` code left. I'll drop this commit.